src/Sarif.UnitTests/SarifVersionOneToCurrentTests.cs

```diff
@@ -1114,5 +1119,282 @@ public void SarifTransformerTests_ToCurrent_BasicResult()
+                                  ]
+                                }
+                              ]
+                            },
+                            {
+                              ""description"": ""Wrap attribute value in double quotes."",
+                              ""fileChanges"": [
+                                {
+                                  ""uri"": ""http://localhost:34420/HtmlFixes.html"",
+                                  ""replacements"": [
+                                    {
+                                      ""offset"": 623,
+                                      ""insertedBytes"": ""Ig==""
+                                    },
+                                    {
+                                      ""offset"": 629,
+                                      ""insertedBytes"": ""Ig==""
+                                    }
+                                  ]
+                                }
+                              ]
+                            }
+                          ]
+		                },
+                        {
+                          ""ruleId"": ""WEB1066"",
+                          ""formattedRuleMessage"": {
+                            ""formatId"": ""default"",
+                            ""arguments"": [
+                              ""DIV""
+                            ]
+                          },
+                          ""suppressionStates"": [ ""suppressedExternally"" ],
+                          ""baselineState"": ""existing"",
+                          ""level"": ""error"",
+                          ""snippet"": ""<DIV id=\""test1\"" xweb:fixindex=\""0\""></DIV>"",
+                          ""locations"": [
+                            {
+                              ""analysisTarget"": {
+                                ""uri"": ""http://localhost:34420/HtmlFixes.html""
+                              },
+                              ""resultFile"": {
+                                ""uri"": ""http://localhost:34420/HtmlFixes.html"",
+                                ""region"": {
+                                  ""startLine"": 24,
+                                  ""startColumn"": 4,
+                                  ""endColumn"": 38,
+                                  ""offset"": 803,
+                                  ""length"": 34
+                                }
+                              }
+                            }
+                          ],
+                          ""fixes"": [
+                            {
+                              ""description"": ""Convert tag name to lowercase."",
+                              ""fileChanges"": [
+                                {
+                                  ""uri"": ""http://localhost:34420/HtmlFixes.html"",
+                                  ""replacements"": [
+                                    {
+                                      ""offset"": 804,
```

Comment:
For consistency: `"offset": 803`.

Reply:
Yes, now that you've explained that the region includes the whole element, this makes sense.